com.amazon.Workspaces.json

```diff
@@ -4,6 +4,7 @@
     "runtime-version": "40",
     "sdk": "org.gnome.Sdk",
     "command": "workspacesclient",
+    "automerge-flathubbot-prs": true,
```

Comment:
Not this json file, there's a `flathub.json` file for flathub specific stuff (flathub != flatpak)